backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCDSParquetSuite.scala

```diff
@@ -91,7 +91,7 @@ class GlutenClickHouseTPCDSParquetSuite extends GlutenClickHouseTPCDSAbstractSui
         |)
         |""".stripMargin
     val result = spark.sql(testSql).collect()
-    assert(result(0).getLong(0) == 791980)
+    assert(result(0).getLong(0) == 791809)
```

Comment:
the value 791809 is the same as the one of vanilla spark ?

Reply:
yes,791809 is the result of vanilla spark